docs/guide/config/preferences.md

```diff
@@ -24,7 +24,100 @@ TODO
 
 ## Input System
 
-TODO
+The Input System tab is used to configure how to handle input devices such
+as mice, styluses, keyboards, and touchscreens.
+
+### Overriding Device Classes
+
+By default, Xournal++ will guess the "class" of each physical input device that
+is recognized by your computer.
+Sometimes the guess will be wrong (e.g., if your stylus is detected as a
+touchscreen), in which case you may need to manually override device class in
+the `Input Devices` section.
+
+Each device recognized by Xournal++ will be displayed in the format of `Name
+(guessed device type)`, one per line.
+The right side of each line will have a dropdown menu that can be used to
+override the device class.
+
+The device classes are described below:
+
+* **Disabled**: do not handle any input from the physical device.
+* **Mouse**: treat the physical device like a mouse, but do not allow keyboard
+  input from the physical device.
+  Supports pressure sensitivity if assigned to a stylus.
+* **Mouse+Keyboard Combo**: similar to mouse, but allow keyboard input.
+  This device class should be used for wireless USB "Mouse + Keyboard Combo"
+  devices, such as popular ones from Logitech.
+* **Pen**: a stylus device tip. Supports pressure sensitivity.
+* **Eraser**: forces all input from the physical device to be used with the
+  Eraser Tool.
+  Supports pressure sensitivity.
```

Comment:
I don't think the eraser is sensitive to pressure.
```suggestion
```

Reply:
You're right. It seems like the device might report pressure sensitivity, but the application does not use the pressure data in the eraser tool logic.